c/alloc.c

```diff
@@ -612,6 +612,7 @@ ptr S_mkcontinuation(s, g, nuate, stack, length, clength, link, ret, winders)
     CONTLINK(p) = link;
     CONTRET(p) = ret;
     CONTWINDERS(p) = winders;
+    CONTATTACHMENTS(p) = attachments;
```

Comment:
As a pick from Travis CI complains, `alloc.c:615:5: error: implicit declaration of function 'CONTATTACHMENTS' is invalid in C99 [-Werror,-Wimplicit-function-declaration]`

Reply:
The build will only work if you cross-compile first with an existing Chez Scheme. See the "When you make a modification to the system that causes the C side to..." note that Bob Burger recently added to "BUILDING", or see the discussion of #233 for an alternate sequence.